src/Logger.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Logger.js
+ * ============
+ * Wrapper around Winston logger capability that will allowus to standardize logging for FTServer more easily.
+ *
+ * For now I am keeping the console logger with default level of info. The log level can be overridden via an environment varaible.
+ * I setup cli logging to keep things consistent with the way we had before, but we could switch to json logging and add a timestamp
+ * by adding the following 
```

Comment:
Is there a missing comment here? What is "the following"?

Reply:
Fixed the comment to refer to the guide